x-pack/plugins/maps/public/actions/data_request_actions.ts

```diff
@@ -119,10 +121,11 @@ function getDataRequestContext(
   getState: () => MapStoreState,
   layerId: string,
   forceRefreshDueToDrawing: boolean,
-  isForceRefresh: boolean
+  isForceRefresh: boolean,
+  isEditingFeatures: boolean
```

Comment:
isEditingFeatures does not need to be a parameter of getDataRequestContext. You can just set isEditingFeatures in getDataRequestContext because you have getState and layerId